python/samples/demos/workflow_evaluation/create_workflow.py

```diff
@@ -0,0 +1,482 @@
+
+Agents:
+1. Travel Agent - Main coordinator (no tools to avoid thread conflicts)
+2. Hotel Search - Searches hotels with tools
+3. Flight Search - Searches flights with tools  
+4. Activity Search - Searches activities with tools
+5. Booking Information Aggregation - Aggregates hotel & flight booking info
+6. Booking Confirmation - Confirms bookings with tools
+7. Booking Payment - Processes payments with tools
+"""
+
+import asyncio
+import os
+from collections import defaultdict
+
+from dotenv import load_dotenv
+
+from agent_framework import (
+    AgentExecutorResponse,
+    AgentRunUpdateEvent,
+    AgentRunResponseUpdate,
+    ChatMessage,
+    Executor,
+    executor,
+    handler,
+    Role,
+    WorkflowContext,
+    WorkflowBuilder,
+    WorkflowOutputEvent,
+)
+
+from agent_framework.azure import AzureAIClient
+from azure.identity.aio import DefaultAzureCredential
+from azure.ai.projects.aio import AIProjectClient
+
+from _tools import (
+    # Travel planning tools
+    search_hotels,
+    get_hotel_details,
+    search_flights,
+    get_flight_details,
+    search_activities,
+    confirm_booking,
+    check_hotel_availability,
+    check_flight_availability,
+    process_payment,
+    validate_payment_method,
+)
+
+load_dotenv()
+
+
+@executor(id="start_executor")
+async def start_executor(input: str, ctx: WorkflowContext[list[ChatMessage]]) -> None:
+    """Initiates the workflow by sending the user query to all specialized agents."""
+    await ctx.send_message([ChatMessage(role="user", text=input)])
+
+
+class ResearchLead(Executor):
+    """Aggregates and summarizes travel planning findings from all specialized agents."""
+    
+    def __init__(self, chat_client: AzureAIClient, id: str = "travel-planning-coordinator"):
+        # store=True to preserve conversation history for evaluation
+        self.agent = chat_client.create_agent(
+            id="travel-planning-coordinator",
+            instructions=(
+                "You are the Travel Planning Coordinator. Your role is to synthesize information from multiple "
+                "specialized travel agents into a cohesive, actionable travel plan. You receive inputs from: "
+                "hotel search specialists, flight search specialists, activity planners, booking confirmation agents, "
+                "payment processors, and booking information aggregators. Provide a clear, comprehensive travel plan "
+                "that addresses the user's original query with all necessary details including accommodations, "
+                "transportation, activities, and booking status."
+            ),
+            name="travel-planning-coordinator",
+            store=True
+        )
+        super().__init__(id=id)
+
+    @handler
+    async def fan_in_handle(self, responses: list[AgentExecutorResponse], ctx: WorkflowContext[WorkflowOutputEvent]) -> None:
```

Comment:
```suggestion
    async def fan_in_handle(self, responses: list[AgentExecutorResponse], ctx: WorkflowContext[Never, str]) -> None:
```
because this handle doesn't produce any messages, only outputs

Reply:
done